frontend/src/pages/organization/GroupDetailsByIDPage/components/GroupProjectsSection/GroupProjectsSection.tsx

```diff
@@ -0,0 +1,90 @@
+import { faPlus } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+import { createNotification } from "@app/components/notifications";
+import { OrgPermissionCan } from "@app/components/permissions";
+import { DeleteActionModal, IconButton } from "@app/components/v2";
+import { OrgPermissionGroupActions, OrgPermissionSubjects } from "@app/context";
+import { useDeleteGroupFromWorkspace as useRemoveProjectFromGroup } from "@app/hooks/api";
+import { usePopUp } from "@app/hooks/usePopUp";
+
+import { AddGroupProjectModal } from "../AddGroupProjectModal";
+import { GroupProjectsTable } from "./GroupProjectsTable";
+
+type Props = {
+  groupId: string;
+  groupSlug: string;
+};
+
+export const GroupProjectsSection = ({ groupId, groupSlug }: Props) => {
+  const { popUp, handlePopUpOpen, handlePopUpToggle } = usePopUp([
+    "addGroupProjects",
+    "removeProjectFromGroup"
+  ] as const);
+
+  const { mutateAsync: removeProjectFromGroupMutateAsync } = useRemoveProjectFromGroup();
+
+  const handleRemoveProjectFromGroup = async (projectId: string, projectName: string) => {
+    await removeProjectFromGroupMutateAsync({
+      groupId,
+      projectId
+    });
+
+    createNotification({
+      text: `Successfully removed project ${projectName} from the group`,
+      type: "success"
+    });
+
+    handlePopUpToggle("removeProjectFromGroup", false);
+  };
+
+  return (
+    <div className="w-full rounded-lg border border-mineshaft-600 bg-mineshaft-900 p-4">
+      <div className="flex items-center justify-between border-b border-mineshaft-400 pb-4">
+        <h3 className="text-lg font-medium text-mineshaft-100">Group Projects</h3>
+        <OrgPermissionCan I={OrgPermissionGroupActions.Edit} a={OrgPermissionSubjects.Groups}>
+          {(isAllowed) => (
+            <IconButton
+              isDisabled={!isAllowed}
+              ariaLabel="add project"
+              variant="plain"
+              className="group relative"
+              onClick={() => {
+                handlePopUpOpen("addGroupProjects", {
+                  groupId,
```

Comment:
IGNORE PLACEMENT OF COMMENT

The spacing of the columns vary a lot depending on the length of the project name, description, etc. which makes it feel wonky 

<img width="880" height="836" alt="image" src="https://github.com/user-attachments/assets/feef93d0-4461-4eb7-8072-36845ba02fa0" />
<img width="913" height="836" alt="image" src="https://github.com/user-attachments/assets/bb3adccf-812d-407c-bf06-3765c65af34d" />